Modules/_sqlite/module.c

```diff
@@ -63,49 +63,49 @@ pysqlite_get_state(PyObject *Py_UNUSED(module))
     return &pysqlite_global_state;
 }
 
-static PyObject* module_connect(PyObject* self, PyObject* args, PyObject*
-        kwargs)
-{
-    /* Python seems to have no way of extracting a single keyword-arg at
-     * C-level, so this code is redundant with the one in connection_init in
-     * connection.c and must always be copied from there ... */
+// NOTE: This must equal sqlite3.Connection.__init__ argument spec!
```

Comment:
I would love to be able to use the AC clone feature here, but the sqlite3 module is split into multiple C files, so this is not possible ... at the moment.